profile/README.md

```diff
@@ -10,7 +10,7 @@
 
 This is the GitHub organization of Pygame Community, a dedicated online community of pygame fans and users.
 
-We aim to enhance the community around the **[`pygame`](https://www.github.com/pygame/pygame)** Python library by hosting exciting game jams, creating cool community games, and more.
+We aim to enhance the community around the **[`pygame`](https://github.com/pygame-community/pygame-ce)** Python library by hosting exciting game jams, creating cool community games, and more.
 We provide a welcoming space to learn, master and create games with pygame, or to hang out with developers, content creators, gamers, artists, musicians, and a lot of other cool people.
 
-**[Join us on Discord!](https://discord.gg/ZuB2RySPRJ)** | **[YouTube](https://www.youtube.com/channel/UCadZkRAdAfelko4mcD3Uq4Q)** | **[Patreon](https://www.patreon.com/pygamecommunity)**
+**[Join us on Discord!](https://discord.gg/ZuB2RySPRJ)** | **[YouTube](https://www.youtube.com/channel/UCadZkRAdAfelko4mcD3Uq4Q)** | **[Patreon](https://www.patreon.com/pygamecommunity)** | **[Website](https://pyga.me/)**
```

Comment:
I reckon perhaps the website should be placed first?

Reply:
Seems reasonable.